### PR TITLE
OCP 4.12 kernel doesn't support bpf_vprintk()

### PR DESCRIPTION
**Pull request by msherif1234**
Fix issue #74
list of functions index is here https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/bpf.h#L5256 for reference 
Signed-off-by: Mohamed Mahmoud <mmahmoud@redhat.com>

### CODE REVIEW

File: bpf/headers/bpf_helpers.h [hunk 1]
```diff
@@ -257,6 +257,11 @@ enum libbpf_tristate {
...
 
 /* Helper macro to print out debug messages */
-#define bpf_printk(fmt, args...) ___bpf_pick_printk(args)(fmt, ##args)
 
+/* #define DEBUG */
+#ifdef DEBUG
+#define bpf_printk(fmt, args...) ___bpf_pick_printk(args)(fmt, ##args)
+#else
+#define bpf_printk(fmt, args...)
+#endif
```
Comment (martinkennelly) [inline bpf/headers/bpf_helpers.h:266]: isnt that an extra endif?

Comment (msherif1234) [reply bpf/headers/bpf_helpers.h:266]: why ?

Comment (martinkennelly) [reply bpf/headers/bpf_helpers.h:266]: I thought, erroneously that there was an extra endif there.

**Pull request merged**